versions/3.0.md

```diff
@@ -1822,30 +1893,80 @@ X-Rate-Limit-Reset:
 
 #### <a name="exampleObject"></a>Example Object
 
-Allows sharing examples for operation requests and responses. This object can either be a freeform object, array or primitive value.  To represent examples of media types that cannot naturally represented in the OpenAPI definition, a string value can be used to contain the example with escaping where necessary. 
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="exampleSummary"></a>summary | `string` | Short description for the example.
+<a name="exampleDescription"></a>description | `string` | Long description for the example.
+<a name="exampleValue"></a>value | Any | Embedded literal example. The `value` field and `externalValue` field are mutually exclusive. To represent examples of media types that cannot naturally represented in JSON or YAML, a string value can be used to contain the example with escaping where necessary.
+<a name="exampleExternalValue"></a>externalValue | `string` | A URL that points to the literal example. This provides the ability to reference examples that cannot easily be included in JSON or YAML documents.  The `value` field and `externalValue` field are mutually exclusive. 
 
-##### Example Example
+This object can be extended with [Specification Extensions](#specificationExtensions). 
 
-Example representation for application/json media type of a Pet data type:
+In all cases, the example value is expected to be compatible with the type schema 
+for the value that it is accompanying.  Tooling implementations MAY choose to 
+validate compatibility automatically, and reject the example value(s) if they 
+are not compatible.
 
-```json
-  {
-    "name": "Puma",
-    "type": "Dog",
-    "color": "Black",
-    "gender": "Female",
-    "breed": "Mixed"
-  }
-```
+##### Example Object Example
 
 ```yaml
+# in a model
+schemas:
+  properties:
+    name:
+      type: string
+      examples:
+        name:
+          $ref: http://example.org/petapi-examples/OpenApi.json#/components/examples/name-example
+
+# in a request body, note the plural `examples` as the Content-Type is set to `*`:
```

Comment:
It should be. The comment itself doesn't seem in place though because nothing in the example describes a wildcard content type.